src/components/customized-components/ClinicDetailComponent/ClinicDetail.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+// import {}
+
+const ClinicDetailComponent = (props) => {
```

Comment:
change this into {imgSrc, name, serviceArray,...} or pass an object into this, props is confusing